packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -37,9 +37,9 @@ export function registerASA (
   return cp;
 }
 
-export function registerASC (
-  cp: Checkpoint, name: string, info: ASCInfo): Checkpoint {
-  cp.asc.set(name, info);
+export function registerSSC (
+  cp: Checkpoint, name: string, info: SSCInfo): Checkpoint {
+  cp.ssc.set(name, info);
   return cp;
 }
```

Comment:
can we remove this function and use directly:
```
cp.ssc.set(name, info)
```

Reply:
these functions are being used in tests 
https://github.com/scale-it/algorand-builder/blob/0302d0cc4d12ba9245e5d0fcbd072d7a6bb3a111/packages/algob/test/lib/script-checkpoints.ts#L86
will have to change tests somehow that they become independent of these functions

Reply:
I tests we can use them directly as well, right?

Reply:
ok, let me remove this and update tests

Reply:
i thought these functions were doing something related to test but they were only putting values, thanks